### DISCUSSION TITLE
Discussions are now available

### DISCUSSION THREAD

**Discussion by Cyrexxis** [Cyrexxis/RokTracker#65] [Announcements]
I just wanted to let all know that discussions are now available. I think the most interesting categories will be the ideas section and the assistance section for the people that need help.

If you plan on posting in the assistance section please keep in mind that you probably need to post the log of the scanner. Otherwise it will be very difficult to track down the problem. Also read the readme beforehand to be sure that you did everything correct.

**Comment by GravityPython** [comment]
Hi Cyrexxis,
First of all,  well done this is an amazing work that you are doing .
I have been using the tracker on a Pc but also in Virtual Machines (VM). As you may know performance of VM is lower than a PC so sometimes the scanning time for full details goes up to 45 sec per player and usually the scan never reaches the 300 players. I would like to suggest the following :
1- Can you make some changes to use LDplayer instead of Bluestacks, it appears that LDplayer is more stable and less demanding in terms of resources. 
2- Is it possible to execute the kingdom_scanner_console with parameter in the command line , this for  automated execution
3- If have seen that with lower resolution 1027x567 240dpi , kingdom scanner gives accurate results. the idea is make the game play faster at lower resolution. Do you think it can work for the scanner?

Many thanks
Gravity

**Reply by Cyrexxis** [reply]
Hello, I'll respond to the points individually:

1. I think there doesn't need to be much changing, in theory it should work. The only thing that should not work is the automatic port detection.
2. Good point. From time to time I thought about that, but always forgot again later.
3. That would need thorough testing, and I am scared that maybe then the governor ID scanning gets more vulnerable to errors. Additionally, I would need to change the positions for the scan and the scrolling scripts.

In general, I tried other players and for my PC Bluestacks worked the best, the others had way more lag. But maybe I configured them wrong. Thanks for the input.

Edit:
I just downloaded LD Player 9, and it is even lagging just opening the play store. It is also warning me that virtualization is enabled, and I should disable that and restart my PC. However, I need that for WSL2 and Docker for my other development stuff. Probably that was the reason I stuck to Bluestacks.

**Reply by GravityPython** [reply]
Hi,
Thanks for the update.
I have changed to 2 months ago to LDplayer ago for a Rok title giver bot that runs 24/7 on a Win11 VM. It is more stable on a long run than Bluestacks. 
Android emulators are not good friends with virtualization, I am able to run both of them (BS & LD) on my PC but without WSL2.  I can do the test on LDplayer if you want.

Regards
G

**Reply by Cyrexxis** [reply]
Hm, that would be an idea. Like I said, the normal stuff should be working when you use the same resolution/dpi as bluestacks. But I have no idea if the scrolling stuff is working. The easy way to see if it does is to do an alliance or honor scan, because for those the scrolling is needed (for kingdom it is only needed for inactive accounts)

**Reply by ShawnCrocker** [reply]
I'm testing the exe version now with Nox, and I just gave the port.
Overall, Nox is currently the least worst emulator for me but it changes over time.

**Comment by leonyonz** [comment]
just wanna say thanks for this amazing tools :)

**Comment by Chump37** [comment]
The rok.tracker.zip download hyperlink in the latest release page does not work, security certificate for the page seems to be outdated.

**Comment by Syorexia** [comment]
Hi Cyrexxis,
Thank you so much for this tool!
Wish to ask if this works or if can work for call of dragon anyhow? Thank you!

**Comment by Dominic8814** [comment]
Hello guys , i installed the bot and it seems to work fine but the only problem i got is that on my spreedsheet it`s putting just -1 values as shown ! Game is in ENGLISH !!
Can anyone give me some help please ! 

Thank you !!

![image](https://github.com/user-attachments/assets/6e1a5195-01a9-4075-84f3-ff44f1e2f18d)
![image](https://github.com/user-attachments/assets/9bb9ff3f-d9ad-4f09-b72e-d007d928eedb)

**Reply by leonyonz** [reply]
please tell us more step by step about how you run the bot..

**Reply by Dominic8814** [reply]
Thank you for your reply but i managed to make it work , didnt kept the resolution after restart and i double checked it ! Now it works Fine and it`s great ! 

Amazing job guys thank you so much ! Much appreciated !
